include/bluetooth/gatt.h

```diff
@@ -699,6 +699,40 @@ ssize_t bt_gatt_attr_read_cpf(struct bt_conn *conn,
 int bt_gatt_notify(struct bt_conn *conn, const struct bt_gatt_attr *attr,
 		   const void *data, u16_t len);
 
+
+/** @brief Notification complete result callback.
+ *
+ *  @param conn Connection object.
+ */
+typedef void (*bt_gatt_notify_complete_func_t) (struct bt_conn *conn);
+
+/** @brief GATT Notify Value parameters */
+struct bt_gatt_notification_params {
+	/** Characteristic or Characteristic Value attribute. */
+	const struct bt_gatt_attr *attr;
+
+	/** Notification Value callback. */
```

Comment:
btw - why "Value" or "Attribute"... maybe we should keep it lower case (just like in line 720)?